### ISSUE TITLE
STPPaymentCardTextField & STPCardParams compile issues

### ISSUE THREAD

**Issue by mohammdsss1** [stripe/stripe-ios#1726]
## Summary
The App doesn't compile after upgrading Stripe from `20.0.0` to `21.0.1`

## Code to reproduce
```
private let cardTextField = STPPaymentCardTextField()
let card = STPCardParams()
card.cvc = cardTextField.cvc
card.number = cardTextField.cardNumber
card.expYear = cardTextField.expirationYear
card.expMonth = cardTextField.expirationMonth
```

## iOS version
iOS 12.0 and above

## Installation method
Cocoapods

## SDK version
`21.0.1`

![Screen Shot 2020-11-30 at 8 40 17 PM](https://user-images.githubusercontent.com/7831855/100650223-44383f00-334c-11eb-85ab-2e29b8360003.png)

**Comment by davidme-stripe** [comment]
Hi, thanks for filing! We'll fix this soon, but for now I'd recommend getting `cardTextField.cardParams` and using the data from there instead. (e.g. `cardTextField.cardParams.number`)

**Comment by davidme-stripe** [comment]
This is fixed in version `21.1.0`.
